test/csgo_stats/logs/parser_test.exs

```diff
@@ -159,6 +159,30 @@ defmodule CsgoStats.Logs.ParserTest do
                 penetrated: false
               }} = Parser.parse(line)
 
+      knifed =
+        "\"Niles<16><BOT><TERRORIST>\" [632 585 91] killed \"Albert<23><BOT><CT>\" [600 555 154] with \"knife_t\""
+
+      assert {:ok,
+              %Events.Killed{
+                killer: %{username: "Niles"},
+                killed: %{username: "Albert"},
+                weapon: "knife_t",
+                headshot: false,
+                penetrated: false
+              }} = Parser.parse(knifed)
+
+      knifed =
```

Comment:
Not really a knife kill 🤷‍♂ I also think this test is slightly redundant when the test above already tests for underscore. So maybe just remove this test? Or you can keep it if you can find another name for it.

Moving forward, the parser tests should be a great place for trying out some property-based testing, but I'll hold the horse on that one right now :)

Reply:
Whoops haha. Changed the variable to silencer. Kept it just to have a different weapon check.

Agree, could be fun to try out for this.